tests/by-util/test_sort.rs

```diff
@@ -531,31 +531,31 @@ fn test_keys_invalid_field() {
     new_ucmd!()
         .args(&["-k", "1."])
         .fails()
-        .stderr_only("sort: failed to parse key `1.`: failed to parse character index ``: cannot parse integer from empty string");
```

Comment:
While you're at it, could you fix the other places where I used backticks for quoting in `sort` as well?